code/modules/mob/dead/observer/observer.dm

```diff
@@ -602,12 +602,8 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 /mob/dead/observer/verb/view_manifest()
 	set name = "View Crew Manifest"
 	set category = "Ghost"
-
-	var/dat
-	dat += "<h4>Crew Manifest</h4>"
-	dat += GLOB.data_core.get_manifest(OOC = TRUE)
-
-	src << browse(dat, "window=manifest;size=370x420;can_close=1")
+	var/datum/ui_module/ghost_crew_manifest/CM = new()
+	CM.ui_interact(usr)
```

Comment:
Define this as a global and interact with it, similar to how the ERT menu is done